js/company/company.js

```diff
@@ -0,0 +1,18 @@
+angular.module("modelCompany", [])
+    .factory("localModelCompany", function(){
+
+    function Company(nazwa, adres, nip, nr_konta){
+        this.nazwa = nazwa;
+        this.adres = adres;
+        this.nip = nip;
+        this.nr_konta = nr_konta;
+    }
+
+    return{
+        createCompany: function(nazwa, adres, nip, nr_konta){
+            var company = new Company(nazwa, adres, nip, nr_konta);
+            return company;
```

Comment:
esli w jednej linii cos deklarujesz, a w kolejnej masz tylko "return obiekt", to nie deklaruj tego jako zmienna, tylko od razu rob return:

return new Company(nazwa, adres, nip, nr_konta);

zawsze jedna linia mniej :)